docker_client.go

```diff
@@ -155,6 +166,20 @@ func getContainers(client *docker.Client) ([]*RuntimeContainer, error) {
 				ReadWrite: container.VolumesRW[k],
 			}
 		}
+		if apiInfo != nil {
```

Comment:
I think this could be done once before getting the containers.  All the containers can just have point to the same reference since it's all the same.